rootfs/usr/bin/lnms

```diff
@@ -1,3 +1,14 @@
 #!/bin/sh
 
-yasu librenms:librenms php -f /opt/librenms/lnms "$@"
+currUser=`ps -o user= -p $$`
+currGroup=`ps -o group= -p $$`
+
+if [[ "$currUser" == "$LIBRENMS_USER" ]]; then
+  if [[ "$currGroup" == "$LIBRENMS_GROUP" ]]; then
+    php -f /opt/librenms/lnms "$@"
+  else
+    yasu librenms:librenms php -f /opt/librenms/lnms "$@";
+  fi
+else
+  yasu librenms:librenms php -f /opt/librenms/lnms "$@";
+fi
```

Comment:
Should be enough:

```suggestion
if [ "$(id -u -n)" = "librenms" -a "$(id -g -n)" = "librenms" ]; then
  php -f /opt/librenms/lnms "$@"
else
  yasu librenms:librenms php -f /opt/librenms/lnms "$@"
fi
```

Reply:
@crazy-max `id` isn't available on all distributions, which is why I used `ps`

Reply:
This script is not intended to be used outside the container. What's your use case?

Reply:
@crazy-max exactly as you state; however, I didn't want to limit you or anyone else in case the container changed.